gcp_variant_transforms/pipeline_common.py

```diff
@@ -54,6 +57,12 @@
     '--range_partitioning=start_position,0,{TOTAL_BASE_PAIRS},{PARTITION_SIZE} '
     '--clustering_fields=start_position,end_position '
     '{FULL_TABLE_ID} {SCHEMA_FILE_PATH}')
+_BQ_DELETE_TABLE_COMMAND = 'bq rm -f -t {FULL_TABLE_ID}'
+_GCS_DELETE_FILES_COMMAND = 'gsutil -m rm -f -R {ROOT_PATH}'
+_BQ_LOAD_JOB_NUM_RETRIES = 5
+_MAX_NUM_CONCURRENT_BQ_LOAD_JOBS = 5
```

Comment:
So this is BQ limitation right? Not affected by the count of cores or the count of workers invoked?